libs/klv/src/main/java/org/codice/ddf/klv/KLV.java

```diff
@@ -0,0 +1,558 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.klv;
```

Comment:
It'd also be nice to structure the package a bit. I like to have the things that you'd expect someone to use closer to the top level package. So maybe your `KlvDecoder` class is at the top level and then there would be further packages under `org.codice.ddf.libs.klv` to partition out the other classes. So maybe all the KlvNumber type classes in their own packages and so forth. This makes it easier for someone who is trying to use this library for the first time to understand what classes they should be focusing on, instead of looking in this package and seeing 20 classes.